password-manager-comparison.js

```diff
@@ -119,6 +119,7 @@ var passwordManagerTable = function() {
     k: "LastPass's YubiKey support uses YubiCo's proprietary OTP protocol rather than the more secure U2F (a.k.a., FIDO2, WebAuthn) that the other password managers use. It's probably good enough, but this is nevertheless an important distinction.",
     l: "You can use backup codes or your backup telephone number to recover from a lost 2fa device. If you didn't save backup codes and you didn't set up a recovery phone number or it has changed, then you may be out of luck, or you may be able to contact Dashlane support and ask them to disable your 2fa.",
     m: "This functionality is on Keeper's roadmap and expected to be delivered early in 2020.",
+    n: "NFC-based YubiKey Neo confirmed. Unsure about direct USB (i.e. USB Type C) models. This is likely using YubiCo's proprietary OTP protocol similar to LastPass."
```

Comment:
Please change the last sentence here to read, "Bitwarden supports both the more secure U2F and the less secure Yubico OTP in the browser, but if you want to use your YubiKey with Bitwarden on Android, you will have to use Yubico OTP there; Bitwarden does not yet support U2F on Android."

I'm a bit reluctant to muddy things like this and I wish Bitwarden had chosen one or the other rather than supporting both Yubico OTP and U2F, but regardless if you make the change proposed in the previous paragraph I'll accept the PR. Thanks!

Reply:
Updated the language 👍